contrib/dunst_xr_theme_changer.sh

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env bash
+###############################################################################
+#
+# This script creates a dunst user configuration file in
+# $XDG_CONFIG_HOME/dunst/ folder, changing dunst basic teeming options
+# according to your current X resources color palette.
+#
+# It gets your current user configuration file
+# -$HOME/.conf/dunst/dunstrc, which is copied from the default
+# config file in /usr/share/dunst/dunstrc if it does not exist- and
+# changes the defined attributes -declared further on in the script as
+# $theme_attr_dict dictionary- to the new ones. Then it dumps the new
+# configuration to $user_xr_color_conf file.
+#
+###############################################################################
+
+set -e
+
+script_name="$(basename $0)"
+
+# Function to get resource values
+xrdb_get () {
+    output=$(xrdb -q | grep -e "$1" | cut -f2)
+    default="$2"
+    printf '%s' "${output:-$default}"
+}
+
+# Default config dir and file
+example_conf_dir="/usr/share/dunst"
+example_conf="$example_conf_dir/dunstrc"
+
+if [ -d "/usr/share/dunst" ]; then
+    # Archlinux default dir and file
+    example_conf_dir="/usr/share/dunst"
+
+    if [ -f "$example_conf_dir/dunstrc" ]; then
+        example_conf="$example_conf_dir/dunstrc"
+    else
+        printf 'Could not find the example config file in %s.
+               \nPlease, change \$example_conf variable in the script' "$example_conf_dir"
+        exit 1
+    fi
+
+elif [ -d "/usr/share/doc/dunst" ]; then
+    # Debian/Ubuntu default dir
+    example_conf_dir="/usr/share/doc/dunst"
+
+    if [ -f "$example_conf_dir/dunstrc.example.gz" ]; then
+        # Ubuntu <= 17.10 and Debian <= 1.2.0-2 default file:
+        example_conf="$example_conf_dir/dunstrc.example.gz"
+    elif [ -f "$example_conf_dir/dunstrc.gz" ]; then
+        # Ubuntu >= 18.04 and Debian >= 1.3.0-2 default file:
+        example_conf="$example_conf_dir/dunstrc.gz"
+    else
+        printf 'Could not find the example config file in %s.
+               \nPlease, change \$example_conf variable in the script' "$example_conf_dir"
```

Comment:
As you're using single quotes in the format string, you'll print `\$example_conf` and not solely `$example_conf`.